src/vsr/replica_test.zig

```diff
@@ -964,49 +964,6 @@ test "Cluster: sync: sync, bump target, sync" {
     try TestReplicas.expect_sync_done(t.replica(.R_));
 }
 
-test "Cluster: sync: R=2" {
```

Comment:
Hm, can we keep the test in? Even if there's no longer any special code needed, this is still an interesting case to test

Reply:
Sure! I made a couple tweaks to it:
- The test doesn't actually trigger state sync anymore. (R=2 clusters _can't_ trigger state sync anymore, not counting standbys.)
- Prepare more ops into the new checkpoint, to emphasize the pipeline.